pdfrw/objects/pdfstring.py

```diff
@@ -316,6 +329,11 @@ class PdfString(str):
     indirect = False
 
 
+    # The byte order mark, in both bytes and unicode form
+
+    bytes_bom = codecs.BOM_UTF16_BE
+    uni_bom = bytes_bom.decode('latin-1')
```

Comment:
Ah, apologies, I understand now. So this isn't checking for the BOM in the unicode string, it's checking whether, if we used PdfDocEncoding to convert the string to bytes, the first two bytes would be the BOM. That makes sense, though I'm not quite sure what to call it. `accidental_bom_pdfdocenc`?